components/host-dashboard/AddFundsModal.js

```diff
@@ -252,15 +256,20 @@ const AddFundsModal = ({ host, collective, ...props }) => {
                 fromAccount: buildAccountReference(values.fromAccount),
                 account: buildAccountReference(values.account),
               },
-            }).then(() => {
-              setFundDetails({
-                showPlatformTipModal: true,
-                fundAmount: values.amount,
-                description: values.description,
-                source: values.fromAccount.name,
+            })
+              .then(() => {
+                setFundDetails({
+                  showPlatformTipModal: true,
+                  fundAmount: values.amount,
+                  description: values.description,
+                  source: values.fromAccount.name,
+                });
+                actions.setSubmitting(false);
+              })
+              .catch(error => {
+                actions.setSubmitting(false);
```

Comment:
I recommend returning a promise from `onSubmit` (or simpler, use an async function). That way you don't have to set the `submitting` flag by yourself.

Reply:
Thanks for the tip. Done. 👍🏽